meta-refkit/recipes-ros/cmusphinx/sphinxbase_0.8.bbappend

```diff
@@ -0,0 +1,3 @@
+DEPENDS = "libsndfile1 alsa-lib libsamplerate0"
```

Comment:
Shouldn't this bbappend be in `meta-refkit-industrial`? This might be worth doing an upstream patch for a `PACKAGECONFIG[python]` or similar.